### PR TITLE
Bump all versions to latest and add in munit

### PR DESCRIPTION
**Pull request by ckipp01**
I saw that a couple people asked for a gitpod that was up to date, so I figured this was the closest one we had 😆. So I updated everything to the latest. The newest scalatest had a small change, so I made the opinionated choice to just move to munit. Hope that's ok.

Also, this is totally fine if you want to keep it, but what do you think about maybe having something like this under `scalameta/metals-gitpod` or something, and then we just make sure we keep it up to date as we do releases? I'm happy to help with it. wdyt?

### CODE REVIEW

**Comment by tgodzik** [comment]
> Also, this is totally fine if you want to keep it, but what do you think about maybe having something like this under `scalameta/metals-gitpod` or something, and then we just make sure we keep it up to date as we do releases? I'm happy to help with it. wdyt?

Och yeah, I initially I was keeping it here to experiment with a number of features and it seemed highly experimental. We can for sure move it to scalameta, maybe also add some documentation like `Online IDEs`, which we can add github's codespaces later.

File: .gitpod.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ image: openjdk:8
 vscode:
   extensions:
     - scala-lang.scala@0.3.9:kklqw+c/dNRmtTU8B5repw==
-    - scalameta.metals@1.9.0:PtE/gdeE1YPL6It4tEfqpg==
+    - scalameta.metals@1.9.6:r4TvVGu3A2++o0cbEXt2Ig==
```
Comment (tgodzik) [inline .gitpod.yml:5]: Should be fine, I wonder how does the integration with openvsx can handle it. I would be cool to have something like `scalameta.metals@latest`, though I think openvsx might work only per user.

File: project/.bloop/sbt-workspace-build.json [hunk 2]
```diff
@@ -1,1257 +0,0 @@
-{
```
Comment (tgodzik) [inline project/.bloop/sbt-workspace-build.json:1]: Ouch, that was commited :fearful:

**Review by tgodzik — APPROVED** [review]
Thanks for updating it! We can for sure move it to scalameta, We might need to authorize gitpod to do prebuilds and probably fix the config a little, to have docker images build for each commit.

File: .gitpod.yml (hunk 1 above)
Comment (ckipp01) [reply .gitpod.yml:5]: Yea I have no idea how to actually get this hash. I had to start it, uninstall it, and then install it again to get the latest, and then the has was updated and I just grabbed it. I'm assuming there is an easier way than that to do it.

File: project/.bloop/sbt-workspace-build.json (hunk 2 above)
Comment (ckipp01) [reply project/.bloop/sbt-workspace-build.json:1]: haha whoops.

**Pull request merged**